front-end/src/pages/CreateEvent.jsx

```diff
@@ -1,330 +1,104 @@
-import { getInterestCategories } from "../utilities/InterestCategoriesUtilities";
-import { postEventDetails } from "../utilities/EventUtilities";
-import LocationSearchMap from "../components/LocationSearchMap";
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Container, Button } from 'react-bootstrap';
+import EventForm from '../components/EventForm';
+import { postEventDetails, timeZoneAbbreviations } from '../utilities/EventUtilities';
+import { getInterestCategories } from '../utilities/InterestCategoriesUtilities';
 
 export default function CreateEvent() {
-  // set const for useNavigate
-  const navigate = useNavigate()
-  // set interest categories for users to select from 
+  const navigate = useNavigate();
   const [interestCategories, setInterestCategories] = useState([]);
-  // set all event details useState
-  // event title 
-  const [title, setTitle] = useState("");
-  // event start date/time
-  const [eventStart, setEventStart] = useState("");
-  // event end date/time
-  const [eventEnd, setEventEnd] = useState("");
-  // time zone options below in code
-  const [timeZone, setTimeZone] = useState("");
-  // event type = In-person or Virtual
-  const [eventType, setEventType] = useState("In-Person");
-  // event virtual link if a virtual event (ex. - user will input their zoom link)
-  const [virtualEventLink, setVirtualEventLink] = useState(null);
-  // event in-person venue ex-"Downtown Park Center"
-  const [eventVenue, setEventVenue] = useState("");
-  // event details text
-  const [description, setDescription] = useState("");
-  // event category for search functionality (only one cat per event)
-  const [category, setCategory] = useState("");
-  const [eventPhoto, setEventPhoto] = useState("");
-  const [photoPreview, setPhotoPreview] = useState("");
-  // eventVenueAddress = full address "123 Example St, City, St Zip"
-  const [eventVenueAddress, setEventVenueAddress] = useState("");
-  // eventLocation format = "city, state"
-  const [location, setLocation] = useState("");
-  // eventCoordinates = "latitude, longitude"
-  const [eventCoordinates, setEventCoordinates] = useState("")
+  const [title, setTitle] = useState('');
+  const [eventStart, setEventStart] = useState('');
+  const [eventEnd, setEventEnd] = useState('');
+  const [timeZone, setTimeZone] = useState('');
+  const [eventType, setEventType] = useState('In-Person');
+  const [virtualEventLink, setVirtualEventLink] = useState('');
+  const [eventVenue, setEventVenue] = useState('');
+  const [eventVenueAddress, setEventVenueAddress] = useState('');
+  const [description, setDescription] = useState('');
+  const [category, setCategory] = useState('');
+  const [eventPhoto, setEventPhoto] = useState('');
+  const [photoPreview, setPhotoPreview] = useState('');
+  const [location, setlocation] = useState('');
+  const [eventCoordinates, setEventCoordinates] = useState('');
```

Comment:
it looks like you got rid of the comments on the merge here

Reply:
Will re-add!